app/models/post.rb

```diff
@@ -3,6 +3,9 @@ class Post < ApplicationRecord
   has_many :likes
   has_many :comments
 
+  validates :title, presence: true, length: { maximum: 250 }
+  validates :comments_counter, :likes_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
```

Comment:
✅ CommentsCounter must be an integer greater than or equal zero 👍🏻 
✅ LikesCounter must be an integer greater than or equal zero 👍🏻